web/src/components/vis/BoxPlot.vue

```diff
@@ -70,129 +90,181 @@ export default {
         }
       });
       return [min, max];
-    },
```

Comment:
Could we replace this function with a call to [`d3.extent`](https://github.com/d3/d3-array/blob/master/README.md#extent)?

Reply:
Then again maybe not if you will be adding back in grouping, which complicates this function.

Reply:
I would prefer extent if possible.  I'm trying to combine `extent` with `lodash.flatMap` which I think will read better.  I think that should do what you're asking for.